Makefile

```diff
@@ -13,18 +24,200 @@ sars-cov-2-images:
+	minikube image build --build-opt=build-arg=pathogen=${SARS_COV_2} -t ${DOCKER_IMAGE_URI_PATH}/${PANGOLIN_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.pangolin .
 
-build_local_images: build_sars_cov_2_local
-	docker build --build-arg pathogen=${SARS_COV_2} -t ncov2019-artic-nf-illumina:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.ncov2019-artic-nf-illumina .
-	# docker build --build-arg pathogen=${SARS_COV_2} -t ncov2019-artic-nf-nanopore:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.ncov2019-artic-nf-nanopore .
-	# docker build --build-arg pathogen=${SARS_COV_2} -t pangolin:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.pangolin .
+	docker build --build-arg pathogen=${SARS_COV_2} -t ${DOCKER_IMAGE_URI_PATH}/${NCOV_ONT_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.${NCOV_ONT_DOCKER_IMAGE_NAME} .
+	minikube image load ${DOCKER_IMAGE_URI_PATH}/${NCOV_ONT_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+	docker build --build-arg pathogen=${SARS_COV_2} -t ${DOCKER_IMAGE_URI_PATH}/${NCOV_ILLUMINA_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.${NCOV_ILLUMINA_DOCKER_IMAGE_NAME} .
+	minikube image load ${DOCKER_IMAGE_URI_PATH}/${NCOV_ILLUMINA_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+
+	# docker build --build-arg pathogen=${SARS_COV_2} -t ${DOCKER_IMAGE_URI_PATH}/${PANGOLIN_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.pangolin .
+	# minikube image load ${DOCKER_IMAGE_URI_PATH}/${PANGOLIN_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+
+reload_minikube:
+	kubectl delete -f minikube/pipelines/sars-cov-2.yaml
+	sleep 35
+	minikube image rm ${DOCKER_IMAGE_URI_PATH}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+	make build_minikube_local
+	kubectl apply -f minikube/pipelines/sars-cov-2.yaml
 
 shell_local: build_sars_cov_2_local
 	docker run \
 	--rm \
 	-it \
+	--volume ${PWD}/app:/app \
+	${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
+	bash
+
+
+mounted_shell_local: build_sars_cov_2_local
+	docker run \
+	--rm \
+	-it \
+	--volume ${PWD}/app:/app \
+	--volume ${PWD}/local_test/:${CONTAINER_TEST_DATA_PATH} \
 	${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
 	bash
 
+
+# Running this requires commenting/disabling nextflow.config
+test_fastq_local_ont: build_sars_cov_2_local
+	docker run \
+	--rm \
+	--volume ${PWD}/app:/app \
+	--volume ${PWD}/local_test/:${CONTAINER_TEST_DATA_PATH} \
+	${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
+	nextflow \
+		run ./main_new.nf \
+		-c /app/nextflow.local.config \
+		--run 61c06b0a-e5e8-4dbf-8bb0-729cce46a224 \
+		-params-file ${ONT_TEST_DATA_PATH}settings.json \
+		--config-path ${ONT_TEST_DATA_PATH}
+		--output_path ${TEST_OUTPUT_LOCAL}/ont
+
+# Running this requires commenting/disabling nextflow.config
+test_fastq_local_illumina: build_sars_cov_2_local
+	docker run \
+	--rm \
+	--volume ${PWD}/app:/app \
+	--volume ${PWD}/local_test/:${CONTAINER_TEST_DATA_PATH} \
+	${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
+	nextflow \
+		run ./main_new.nf \
```

Comment:
`main.nf`? There is no `main_new.nf` in the repo.

Reply:
Ah yes I was using main_new while developing and have now switched to make the old one main_old and the new one is just main. I've changed all references to it now.

Reply:
Also note the comment above it - you need to comment everything in nextflow.config to make it work, there's no way to make nextflow 'ignore' a config file that I can find. It will still not complete, but you can add a return statement before ncov and it'll run that far allowing you to to test the fastq processing